front-end/webpack.config.js

```diff
@@ -1,4 +1,4 @@
-const path = require('path');
+var path = require('path');
```

Comment:
I'm curious about this var vs. const!

Reply:
I don't know where that came from - haven't touched the webpack! I pulled from development and there were two path declarations so I deleted one. I can fix that.

Reply:
Oh, no problem! Just go with whichever is the most recent incoming change, then~